mongorestore/mongorestore_test.go

```diff
@@ -69,6 +70,57 @@ func TestMongorestore(t *testing.T) {
 	}
 
 	Convey("With a test MongoRestore", t, func() {
+		// As specified in TOOLS-2363, the --db and --collection options
+		// are well-defined only for restoration of a single BSON file
+		Convey("and proper warning is issued if --db and --collection are not supported", func() {
```

Comment:
I don't think this is the best spot for these tests. Look at the other tests in this file, i.e. `TestDeprecatedIndexOptions`. Perhaps you can model your new tests off of that. You aren't using any of the setup code in the outer convey block so these tests probably shouldn't be here. My suggestion is to create a new test function, something like `TestDeprecatedDbAndCollectionOptions`, and move these convey blocks there.

Reply:
Fair point. Consider it done!